### ISSUE TITLE
For queries with no results VDMS returns an error

### ISSUE THREAD

**Issue by mwahle** [IntelLabs/vdms#59]
Instead of an error VDMS should return an empty result set:
```
db.query([{'AddEntity': {'class': 'Test2', 'properties': {'x': 1}}}])
Out[6]: ([{'AddEntity': {'info': '', 'status': 0}}], [])

db.query([{"FindEntity": {'class': 'Test2', 'constraints': {'x': ['==', 2]}}}])
Out[7]: 
([{'FailedCommand': 'Transaction',
   'info': 'Failed PMGDTransaction',
   'status': -1}],
[])
```

**Comment by crstrong** [comment]
I believe this is the same as #36

**Comment by vishakha041** [comment]
Fixed in 45f651f
